airbyte-workers/src/main/resources/dbt_transformation_entrypoint.sh

```diff
@@ -34,27 +34,28 @@ if [[ -f "${CWD}/bq_keyfile.json" ]]; then
   cp "${CWD}/bq_keyfile.json" /tmp/bq_keyfile.json
 fi
 
-. $CWD/sshtunneling.sh
-openssh $CWD/ssh.json
+# shellcheck source=./airbyte-workers/src/main/resources/sshtunneling.sh
+. "$CWD/sshtunneling.sh"
+openssh "$CWD/ssh.json"
 trap 'closessh' EXIT
 
 # Add mandatory flags profiles-dir and project-dir when calling dbt when necessary
 case "${CONTAINS_PROFILES_DIR}-${CONTAINS_PROJECT_DIR}" in
   true-true)
-    echo "Running: dbt $@"
-    dbt $@ --profile normalize
+    echo "Running: dbt $*"
```

Comment:
Since this is printing output, expand the array into a string. But in the actual dbt call keep it as an array